app/assets/stylesheets/modules/_family.scss

```diff
@@ -131,3 +127,66 @@
   }
 }
 
+.family-names {
+  text-align: center;
+  background-color: tint(gray, 60);
+  border-radius: $base-border-radius;
+  margin: em(10) em(80) em(30);
+  padding: em(20) 0;
+}
+
+p {
+  color: gray;
+  font-family: "Courier";
+  font-size: em(20);
+  height: em(25);
+  margin: 0 auto;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+p:nth-child(1) {
+  animation: type 9s steps(37, end);
+  animation-iteration-count: infinite;
+  -webkit-animation-delay: 2s;
+  border-right: 1px solid gray;
+  margin-bottom: em(10);
+  width: em(380);
+}
+
+p:nth-child(2) {
+  animation: type-b 8s steps(32, end);
+  animation-iteration-count: infinite;
```

Comment:
Properties should be ordered animation, -webkit-animation-delay, animation-iteration-count, border-right, width